src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -21,19 +23,135 @@ export interface AssetToBuySection {
   id: AssetToBuySectionId;
 }
 
+const MAX_UNVERIFIED_RESULTS = 8;
+const MAX_VERIFIED_RESULTS = 50;
```

Comment:
constants.ts maybe? Fine to leave these here too

Reply:
Prefer to keep them here since they're not needed elsewhere and I think it's clearer for these to be visible in the context of this file.